docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: PROXY protocol
+description: How to securely configure PROXY protocol usage with Teleport
+---
+<Admonition type="note" scope={["cloud", "team"]}>
+Users of Teleport Cloud don't need to manage PROXY protocol setting manually. It's set to `on` automatically 
+and can't be changed, since cloud Proxy/Auth services always run behind L4 load balancer with enabled PROXY protocol. 
+</Admonition>
+
+## What is PROXY protocol
+
+The PROXY Protocol is a prefix added to the TCP connection containing information about the client IP. 
+It is most commonly used when networking setup includes a Layer 4 (L4) load balancer between the user and the endpoint service,
+like Teleport Proxy/Auth.
+L4 load balancers, by design, do not retain the original client's IP address and port when forwarding the connection and 
+PROXY Protocol allows to overcome this problem by adding the client's original IP address and port before the actual TCP stream.
+
```

Comment:
Before the configuring proxy protocol section, I wanted to see a security-like section where we mention that users that don't run Teleport behind a L4 LB with PROXY PROTOCOL enabled MUST disable proxy protocol because an attacker can send the proxy line and trick teleport into using a incorrect IP in Teleport/bypass IP pinning if proxy_protocol: on is incorrectly configured

Reply:
I think `proxy_protocol: on` is not a problematic mode here, since users probably will notice this misconfiguration from the symptoms of nothing working at all :) (PROXY headers not coming in regular usage -> connections rejected) 
But yeah, it's a problem if `proxy_protocol` is unspecified. I've added security section.